learn/tanssi/external-security-providers/symbiotic.md

```diff
@@ -110,20 +112,133 @@ flowchart LR
     tanssi -- Block Production Services--> Networks
     tanssi -- Security--> Networks
     tanssi -- Data Retrievability--> Networks
+
+    class Symbiotic custom-container
 ```
 
-### Slashing and Rewards {: #slashing-rewards }
+### Rewards {: #rewards }
+
+Well-behaved operators and restakers are rewarded for their participation with TANSSI tokens. The reward process consists of two main phases: [Reward Distribution Phase](#reward-distribution-phase) and [Reward Claiming Phase](#reward-claiming-phase).
+
+#### **Reward Distribution Phase** {: #reward-distribution-phase }
+
+The reward distribution phase calculates and allocates rewards through five key steps involving operators, restakers, and smart contracts. The steps are:
+
+1. Reward Calculation - Tanssi calculates rewards based on the activity of operators and stakers and then creates a [Merkle root](https://en.wikipedia.org/wiki/Merkle_tree). This Merkle root is a cryptographic fingerprint that summarizes the reward allocations, indicating who receives what. Stakers are rewarded according to their stake in each vault
+2. Cross-Chain Messaging - reward allocation data is sent using [XCM](https://docs.moonbeam.network/builders/interoperability/xcm/overview/) (Cross-Consensus Messaging), a standardized protocol for blockchain communication. [Snowbridge](https://wiki.polkadot.network/docs/learn-snowbridge) acts as a trustless bridge between Tanssi and Ethereum
+3. Gateway - once the message is relayed to the `Gateway` contract, this contract serves as Tanssi's authorized entry point on Ethereum for the Snowbridge bridge
+4. Middleware - the `Gateway` forwards the data to the [`Middleware`](https://github.com/moondance-labs/tanssi-symbiotic/blob/main/src/contracts/middleware/Middleware.sol), which is responsible for various tasks, including passing the information to the `OperatorReward` contract
+5. OperatorRewards - this is the final destination for the data. The [`OperatorRewards`](https://github.com/moondance-labs/tanssi-symbiotic/blob/main/src/contracts/rewarder/ODefaultOperatorRewards.sol) contract stores the Merkle tree of the reward allocations and handles the transfer of reward tokens when a claim is made
+
+```mermaid
+sequenceDiagram
+    participant Tanssi Network
+    participant Snowbridge (XCM)
+    participant Gateway Contract
+    participant Middleware
+    participant OperatorRewards
+
+    Tanssi Network->>Tanssi Network: 1. Calculate rewards and Generate Merkle root
+    Tanssi Network->>Snowbridge (XCM): 2. Send XCM message (Merkle root + data)
+    Snowbridge (XCM)->>Gateway Contract: 3. Relay message
+    Gateway Contract->>Middleware: 4. Propagate rewards data
+    Middleware->>OperatorRewards: 5. distributeRewards()
```

Comment:
Not sure if it would look ok with back ticks here, but maybe try it out?  and use your best judgement on what looks best in diagrams